app/models/access_group.rb

```diff
@@ -69,6 +69,12 @@ class AccessGroup < ApplicationRecord
     )
   end
 
+  # all access groups that include any coc_codes
+  scope :for_coc_codes, ->(coc_codes) do
+    quoted = SqlHelper.quote_sql_array(coc_codes, type: :varchar)
+    where("#{quoted_table_name}.coc_codes = #{quoted}")
```

Comment:
Should this be `=` or something more like overlap? The comment says includes any, but I _think_ this is looking for equivalence.

Reply:
fixed, thanks